backend/controllers/gameDataController.ts

```diff
@@ -14,7 +14,7 @@ export const saveGame = asyncHandler(async (req: any, res) => {
 
       if (existingData) {
         existingData.numBits = gameData.numBits;
-        existingData.currencyAmount = gameData.currencyAmount;
+        existingData.currencyAmount = Number(gameData.currencyAmount.toFixed(1));
```

Comment:
This is something put in from the currency bug fix, I have no idea why its showing as different.

Reply:
I think this may actually just be vestigial from then, the actual fix was in the **App.tsx** file: https://github.com/hto-projects/autohack-idle/pull/49/files

So you don't actually need this!